mainwindow.cpp

```diff
@@ -936,7 +937,7 @@ void MainWindow::fillSettingsWithDefaults()
     QStringList possiblePaths;
     QString john;
     // Find john on system path, which is determined by PATH variable
-    QString johnSystemPath = QStandardPaths::findExecutable("john", QStringList());
+    /*QString johnSystemPath = QStandardPaths::findExecutable("john", QStringList());
```

Comment:
We need to search manually if < Qt5. Use something similar to this https://github.com/Tarsnap/tarsnap-gui/commit/5d9157163c5284bb8712ad0f32b9bf2ba5bcc2fa#diff-fd0081fe4276fa1c3d9f87bc40a8d1d1L355 .